sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyServiceClient.java

```diff
@@ -230,7 +230,7 @@ JsonWebKey transformSecretKey(SecretKey secretKey) throws JsonProcessingExceptio
         return service.unwrapKey(vaultUrl, keyName, version, apiVersion, ACCEPT_LANGUAGE, parameters,
             CONTENT_TYPE_HEADER_VALUE, context.addData(AZ_TRACING_NAMESPACE_KEY, KEYVAULT_TRACING_NAMESPACE_VALUE))
             .doOnRequest(ignored -> logger.info("Unwrapping key content with algorithm - {}", algorithm.toString()))
-            .doOnSuccess(response -> logger.info("Retrieved unwrapped key content with algorithm- {}",
+            .doOnSuccess(response -> logger.info("Retrieved unwrapped key content with algorithm - {}",
                 algorithm.toString()))
```

Comment:
nit: Don't call `toString` when passing into a log method, see #19462 for more details